why_postgres.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: 'A new serverless Postgres platform'
+description: "We're announcing the next major evolution for the Xata data platform, serverless Postgres."
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-public-release.png
+  alt: PostgreSQL
+author: Monica Sarbu, Tudor Golubenco
+date: 03-11-2024
+tags: ['announcements']
+published: false
+slug: why-postgres
+---
+
+Tudor's list of alternate titles
+
+- Why are we opening up our Postgres service?
+- Why are we launching a new Postgres platform?
+- A new type of PostgreSQL service
+- Why are we launching a new type of Postgres service
+- Why the world needs a new Postgres cloud service?
```

Comment:
@tsg I moved the alternates into your body so the page could still build a preview.